Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantSedin.cs

```diff
@@ -0,0 +1,49 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Content.Server.Popups;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class PlantSedin : ReagentEffect
```

Comment:
I think that this could be a more generic `ReagentEffect`, instead of being named specifically after Sedin. 
This way new reagents could use this effect, and modify the `SeedRestorationChance` and `PotencyDropAmount` as needed in their own yml.

Reply:
This is a good point. I was just basing it off Robust Harvest's behaviour, which might not have been the best idea. I'll add the code for it and rework Sedin.